### ISSUE TITLE
Cannot compile ‘lookup’ example on main branch

### ISSUE THREAD

**Issue by glts** [NLnetLabs/domain#124]
Looks like there is a problem with code conditional on some Cargo feature.

On current `main` (commit b10fc8b51c56c7ef843088159b2e76e67262c045):

```
cargo run --features resolv --example lookup -- crates.io
```

```
error[E0412]: cannot find type `Bytes` in this scope
   --> src/rdata/rfc5155.rs:618:16
    |
618 | impl Nsec3Salt<Bytes> {
    |                ^^^^^ not found in this scope
    |
help: consider importing one of these items
    |
7   | use bytes::Bytes;
    |
7   | use core::str::Bytes;
    |
7   | use crate::rdata::rfc5155::str::Bytes;
    |
7   | use std::io::Bytes;
    |
      and 1 other candidate
...
...
```

There is probably a way to catch this and similar issues at the CI build stage.

**Comment by glts** [comment]
To be clear, the `resolv` feature cannot be used at all on the main branch, it is broken.

**Comment by partim** [comment]
Apologies for the delay. Looks like the import of `Bytes` in some modules has been conditioned on the wrong feature.

This could be caught in CI, but the matrix of possible features has become quite big, so this would result in a lot of compiling.

**Comment by partim** [comment]
I pushed a fix in [c8742db](https://github.com/NLnetLabs/domain/commit/c8742dbc9723cc9527375623403e363d518beb12). Can you check that that fixes all your issues?

**Comment by glts** [comment]
Thank you. This does it.
